src/MMKV.ts

```diff
@@ -1,4 +1,6 @@
 import { createMMKV } from './createMMKV';
+import { createMockMMKV } from './createMMKV.mock';
```

Comment:
I assume you need to use the `require()` API to import this behind an if statement, otherwise you're including it in production builds

Reply:
We need to include this in production builds in order for the automocking to work right?

Reply:
What I'm saying is you're going to include this file in people's JS bundles unless you use the require API and but it behind an if statement

Reply:
Is that a fact? We can lazy-load it at the bottom if needed?

Reply:
I mean technically it's not really impacting bundle size, it's just a few lines and no extra dependencies